packages/plugins/rubygem-smart_proxy_discovery_image/rubygem-smart_proxy_discovery_image.spec

```diff
@@ -68,20 +93,24 @@ mv %{buildroot}%{gem_instdir}/settings.d/discovery_image.yml.example \
 
 %files
 %dir %{gem_instdir}
+%config(noreplace) %attr(0640, root, foreman-proxy) %{foreman_proxy_settingsd_dir}/discovery_image.yml
+%license %{gem_instdir}/LICENSE
+%{gem_instdir}/bundler.d
```

Comment:
We have `%{foreman_proxy_bundlerd_dir}/%{plugin_name}.rb` below so this surprises me